src/docs/product/error-monitoring/issue-details/index.mdx

```diff
@@ -0,0 +1,46 @@
+
+TO DO Add screenshot here of Issue Details page
+
+Click on an issue on the main Issues page to open the Issue Details page.
+
+This page displays the Details tab when it first opens, but several other tabs are available:
+
+- Activity: A history of activity for the event where team members can share comments.
+- User Feedback: Any feedback collected from users through the <PlatformLink to="/enriching-events/user-feedback/">SDK's user feedback</PlatformLink>.
+- <PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink>: Stored additional files, such as config or log files that are attached to the event.
+- Tags: More detailed information about the tags across all the events included in the issue.
+- Events: A list of all the events included in the issue.
+- Merged Issues: A list of issues that have been merged with this one.
+- Similar Issues: A list of issues that have a similar stack trace.
+
+In the right hand sidebar, [sentry.io](https://sentry.io) reflects how often the issue has occurred in the last 24 hours and the last 30 days, as well as the last time and the first time the issue was seen. If the issue is linked to any GitHub or Jira issues, that's displayed here as well. You can also use this section to link to existing GitHub or Jira issues. Lastly, there is a facet map, which shows the distribution of values for the tags for all events included in the issue. All of these values are based on the environment that you've selected in the Global Header.
+
+In addition, this page provides several other key pieces of information.
+
+## Suspect Commits
+
+If you've configured [suspect commits](/product/releases/suspect-commits/), these are displayed. This allows you to immediately investigate commits that are likely to have generated the issue. The owners of suspect commits can be suggested assignees for an issue as well.
+
+## Tags
+
+Tags are the diagnostic information set by the SDK for the individual event. These are specific to each event in an issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.
+
+## Stack Trace
+
+The Exception section of the page displays the stack trace, which shows you the line of code that all of the events in the issue errored on. It's the most important piece of information that the Sentry grouping algorithm uses to group events into one issue.
+
+## Breadcrumbs
+
+Breadcrumbs provide a history and timeline leading up to the error event. These can include things like HTTP requests, log statements from the console or server, and DOM (document object model) events in JavaScript. You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrumbs</PlatformLink> to make them more useful for debugging.
+
```

Comment:
Need a section that describes/links to Linked Issues from here and why they’re useful

Reply:
Added link in p about right hand sidebar